wodles/aws/tests/aws_utils.py

```diff
@@ -0,0 +1,309 @@
+import os
```

Comment:
Same issue as mentioned [here](https://github.com/wazuh/wazuh/pull/16071#discussion_r1102846050). It would generate an import problem due to `.../wodles/utils.py`